addons/Context/Tweek.Drivers.Context.Couchbase/CouchBaseServiceAddon.cs

```diff
@@ -42,7 +42,11 @@ public void Configure(IServiceCollection services, IConfiguration configuration)
             var couchbaseConfig = configuration.GetSection("Couchbase");
             var contextBucketName = couchbaseConfig["BucketName"];
             var contextBucketPassword = couchbaseConfig["Password"];
+            var serverUrl = couchbaseConfig.GetSection("Url").Get<string>();
             var serverUrls = couchbaseConfig.GetSection("Urls").Get<List<string>>();
+            if (serverUrls == null || serverUrls.Count == 0){
+                serverUrls = new List<string>(){ serverUrl };
```

Comment:
Maybe check that Url exits and not empty?

Reply:
What we can we do if that's the case?

Reply:
It's more like use the old api if "urls" is not used

Reply:
It'll probably get the same exception as before in the ```new Uri(url)``` code